src/settings-ui/Settings.UI/ViewModels/CmdNotFoundViewModel.cs

```diff
@@ -6,17 +6,23 @@
 using System.Diagnostics;
 using System.IO;
 using System.Reflection;
+using AllExperiments;
```

Comment:
???

Reply:
Definitely wrong

Reply:
Was automatically added when I've tried to add the logger...